src/HID-APIs/AbsoluteMouseAPI.h

```diff
@@ -68,9 +68,10 @@ class AbsoluteMouseAPI
 	inline void end(void);
 
 	inline void click(uint8_t b = MOUSE_LEFT);
+	inline void moveTo(int x, int y, signed char wheel = 0, signed char hWheel = 0);
 	inline void moveTo(int x, int y, signed char wheel = 0);
+	inline void move(int x, int y, signed char wheel = 0, signed char hWheel = 0);
```

Comment:
Is the second function even needed? If we provide a default, existing code should function, right?

Reply:
If adding the `hWheel` parameter to the existing method (without overloading) doesn't break existing code elsewhere, then I don't think it would be needed. I believe that's what @WaGi-Coding's PR looked like before my suggestions, and he said those changes were tested. The only difference here is that I also added the `hWheel` parameter to the `moveTo` method.

Reply:
Yeah, pretty sure it was all good before :-D